components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -543,22 +544,28 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
          ! config_GM_spatially_variable_baroclinic_mode
          if (local_config_GM_lat_variable_c2) then
```

Comment:
based on the logic below I don't think the changes will be active. `local_config_GM_lat_variable_c2` is set by `config_GM_closure` so for constant GM this change you made won't do anything.